packages/perseus/src/widgets/transformer.tsx

```diff
@@ -2518,9 +2518,8 @@ class Transformer extends React.Component<Props> {
                 // @ts-expect-error - TS2345 - Argument of type '{ type: "dilation"; center: any; scale: number; }' is not assignable to parameter of type 'PerseusTransformerTransformation'.
                 self.doTransform({
                     type: "dilation",
-                    // NOTE(kevinb): It's hard to convince that this.dilationCircle
```

Comment:
Realized we don't need this comment at all any more, as we've now removed all the flow overrides! No need to convince typescript. :)